erpnext/patches/v6_27/update_two_checkbox_value_for_recurringdoc.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+	for doctype in ('Sales Order', 'Purchase Order', 'Sales Invoice',
+		'Purchase Invoice'):
+		frappe.reload_doctype(doctype)
+		frappe.db.sql('''update `tab{0}` set submit_on_creation=0, notify_by_email=0
+			where is_recurring=1'''.format(doctype))
```

Comment:
the **where** should be **notify_by_email = 1**. we should preserve the data already in the db.